core/src/main/scala/cats/data/package.scala

```diff
@@ -3,6 +3,7 @@ package cats
 package object data {
   type NonEmptyStream[A] = OneAnd[Stream, A]
   type ValidatedNel[+E, +A] = Validated[NonEmptyList[E], A]
+  type IorNel[+B, +A] = Ior[NonEmptyList[B], NonEmptyList[A]]
```

Comment:
@edmundnoble I am going to change it to `type IorNel[+B, +A] = Ior[NonEmptyList[B], A]` to keep consistency with ValidatedNel. I do have a use case for my original proposal but that is when you get to combine Iors with the |+| operator. My main argument to change it is that in the flatMap you get tu accumulate errors on the left side of `Both` so it is the natural usage of it.